config/locales/devise.de.yml

```diff
@@ -0,0 +1,67 @@
+# if me: DO NOT modify this file. if you need to add translations, add them to sv.yml
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
+
+sv:
+  devise:
+    confirmations:
+      confirmed: "Deine E-Mailadresse wurde erfolgreich bestätigt."
+      send_instructions: "Du wirst in einigen Minuten eine E-Mail erhalten mit Anweisungen, wie du deine E-Mailadresse bestätigen kannst."
```

Comment:
If I read this dictionary correctly, "einigen Minutes" would be used with "nach". Otherwise, "einige Minutes" would be used. (-n)

https://www.dict.cc/?s=einigen